RadarSDK/RadarSettings.m

```diff
@@ -333,4 +333,8 @@ + (NSString *)xPlatformSDKVersion {
     return [[NSUserDefaults standardUserDefaults] stringForKey:kXPlatformSDKVersion];
 }
 
++ (BOOL)useOpenedAppConversion {
+    return [[self sdkConfiguration] useOpenedAppConversion];
+}
+
```

Comment:
This should default true I think (first one should happen prior to getting anything synced down from initialize, right?)